health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/UploadDataMapping.js

```
@@ -124,20 +125,18 @@ const reducer = (state, action) => {
-                : i?.[action.t(action?.schemas?.find((i) => i.description === "Facility usage")?.name)] === "Active"
-            ),
-            1,
-            state.rowsPerPage
+      const tempFilter = action.payload?.filter
+        ? state.data?.filter((i) =>
+            action?.currentCategories === "HCM_UPLOAD_USER_MAPPING"
+              ? i?.[action.t(action?.schemas?.find((i) => i.description === "User Usage")?.name)] === "Active"
+              : i?.[action.t(action?.schemas?.find((i) => i.description === "Facility usage")?.name)] === "Active"
           )
-        : getPageData(state.data, 1, state.rowsPerPage);
+        : state.data;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)